### DISCUSSION TITLE
install/commercial/prepare/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1297] [Announcements]
# install/commercial/prepare/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/install/commercial/prepare/

**Comment by panpan0000** [comment]
待补充离线install_prerequisite.sh的步骤

**Reply by jzhupup** [reply]
https://docs.daocloud.io/install/install-tools/#_3

这里有/

然后目前 install_prerequisite.sh 离线有个bug，没有区分arm、amd，等优化好后 这里加一个跳转链接到上述界面

**Reply by panpan0000** [reply]
这里其实还有一个点，就是一些内核参数的调整，比如Ubuntu默认的fs.inotify.max_user_instances只有128

也值得注意。

我在内部有个wiki页面描述
https://dwiki.daocloud.io/pages/viewpage.action?pageId=142518511

**Reply by qiupingdai** [reply]
1. 前置检查增加了存储相关（LVM2 ,落盘..）可以补充进去
2. 现在安装器检查要求所有 global 集群节点都需要有裸盘，需要在不同的节点模式中进行说明

**Reply by jzhupup** [reply]
好的
